edgelet/edgelet-http/src/certificate_manager.rs

```diff
@@ -73,7 +73,7 @@ impl<C: CreateCertificate + Clone> CertificateManager<C> {
             X509::stack_from_pem(cert).with_context(|_| ErrorKind::CertificateConversionError)?;
 
         let mut ca_certs = Stack::new().with_context(|_| ErrorKind::CertificateConversionError)?;
-        for cert in certs.split_off(1) {
+        for cert in certs.split_off(1).into_iter().rev() {
```

Comment:
Feels wrong to reverse it here, since it could potentially be a well-formed PEM provided by the user.

If it's just the quickstart generated one that's backwards, it seems we should fix the quickstart certs generator in libiothsm.

Reply:
Yes, why is this needed? Also please see this https://github.com/Azure/iotedge/blob/50ac495e73ff7b12b47e8227cc1468877aaf9e4e/edgelet/edgelet-http/src/lib.rs#L133. Does this need changing as well?

Reply:
ok, reverted that change. The problem I saw was that the certificate order was mixed up (first one was the certificate of the tls server, then the chain up to the root was reversed). That is not valid according to TLS 1.2 (valid in TLS 1.3, though).